azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunnerManager.java

```diff
@@ -427,6 +431,16 @@ private void cleanOlderProjects() {
     }
   }
 
+  public void deleteDirectory(ProjectVersion pv) throws IOException {
```

Comment:
`ProjectVersion` is currently some data and some utils around projects. I am cleaning it up to be just a data class. I plan to rename it to `ProjectInfo` later.

Deletion and management of data is already done in `FlowRunnerManager`. Also, if you notice, the deleteDirectory method in turn calls a utility method. I agree it can be static. Will change that.